mmdet/models/necks/yolo_spp_pan_neck.py

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2019 Western Digital Corporation or its affiliates.
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+from ..builder import NECKS
+
+
+class DetectionBlock(nn.Module):
+    """Detection block in YOLO neck.
+
+        Let out_channels = n, the DetectionBlock normally contains 5 ConvModules,
+        Their sizes are 1x1xn, 3x3x2n, 1x1xn, 3x3x2n, and 1x1xn respectively.
+        If the spp is on, the DetectionBlock contains 6 ConvModules and
+        3 pooling layers, sizes are 1x1xn, 3x3x2n, 1x1xn,
+        5x5 maxpool, 9x9 maxpool, 13x13 maxpool, 1x1xn, 3x3x2n, 1x1xn.
+        The input channel is arbitrary (in_channels)
+    yolo
+        Args:
+            in_channels (int): The number of input channels.
+            out_channels (int): The number of output channels.
+            conv_cfg (dict): Config dict for convolution layer. Default: None.
+            spp_on (bool): whether to integrate a spp module. Default: False.
+            spp_pooler_sizes (tuple): A set of sizes for spatial pyramid pooling.
+                Default: (5, 9, 13).
```

Comment:
Similar.

Reply:
Should we merge yolo_head and yolo_spp_pan_head? I think keeping them separated would make both file clearer and cause less confusion. They are sufficiently different but it's still a bit of duplicated code. Diff is [here](https://www.diffchecker.com/nLNnV4QN).